contextualized/regression.py

```diff
@@ -416,6 +417,9 @@ def test_step(self, batch, batch_idx):
         self.log_dict({'test_loss': loss})
         return loss
 
+    def predict_from_sample_models(self, X, beta_hat, mu_hat):
```

Comment:
Make private and change name based on model-metamodel convention. `_predict_from_models` or `_model_predict` for example